### PR TITLE
PD-350 Add volumeSnapshotSchedule

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

**Comment by nathan-px** [comment]
@disrani-px I put all of the fields in the example, but I'm pretty confident this needs a simpler example and a separate CRD reference.

File: content/portworx-install-with-kubernetes/storage-operations/create-snapshots/scheduled/_index.md [hunk 1]
```diff
@@ -86,7 +86,54 @@ daily     N/A                10:14PM   N/A                N/A
...
 ```
 
-## Creating a storage class
+## Associate a schedule policy with a StorageClass or a Volume
+
+Once you've defined a schedule policy, you can create and associate `StorageClass` and `Volume` specs with it. Perform one of the following steps, depending on whether you want to schedule snapshots for multiple PVCs or a single volume:
+
+### Create a VolumeSnapshotSchedule
+
+You can create VolumeSnapshotSchedules to backup specific volumes according the a schedule you define.
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/storage-operations/create-snapshots/scheduled/_index.md]: `... according to a schedule...`

File: content/portworx-install-with-kubernetes/storage-operations/create-snapshots/scheduled/_index.md [hunk 2]
```diff
@@ -86,7 +86,54 @@ daily     N/A                10:14PM   N/A                N/A
...
+  * **metadata:**
+    * **name:** with the name of this VolumeSnapshotSchedule policy
+    * **namespace:** the namespace in which this policy will exist
+    * **annotations:**
+      * **portworx/snapshot-type:** with the `cloud` or `local` value, depending on on what environment you want store your snapshots in
+      * **portworx/cloud-cred-id:** with your cloud environment credentials
+      * **stork/snapshot-restore-namespaces:** with other namespaces snapshots taken with this policy can restore to
+  * **spec:**
+    * **schedulePolicyName:** with the name of the schedule policy you defined in the steps above
+    * **suspend:** with a boolean value directing Portworx to create the snapshots defined
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/storage-operations/create-snapshots/scheduled/_index.md]: `with a boolean value specifying if the schedule should be in suspended state` or something similar

File: content/portworx-install-with-kubernetes/storage-operations/create-snapshots/scheduled/_index.md [hunk 3]
```diff
@@ -86,7 +86,54 @@ daily     N/A                10:14PM   N/A                N/A
...
+    * **preExecRule:** with the name of a rule to run before taking the snapshot
+    * **postExecRule:** with the name of a rule to run after taking the snapshot
+    * **reclaimPolicy:** with `retain` or `delete`, indicating what Portworx should do with previously stored snapshots
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/storage-operations/create-snapshots/scheduled/_index.md]: `indicating what Stork should do with the snapshots that were created using the schedule`

This basically specifies if the snapshots created by this schedule should also be deleted when this schedule is deleted.

**Pull request merged**